data/cookbook/web-programming/simple-http-client/00-camlnet.ml

```diff
@@ -0,0 +1,26 @@
+---
+packages:
+- name: "camlnet"
+  version: "4.1.9-2"
+- name: "conf-gnutls"
+  version: "1"
+libraries:
+- netclient
+  nettls-gnutls
+discussion: |
+  - **Understanding `ocamlnet`:** The `ocamlnet` provides a rich set of networking libraries that cover many protocols. It provides some modules with a rich set of parameters. But for simple needs, the `Nethttp_client.Convenience` provides simple functions for basic interaction with the server. Note: These functions don't handle redirections.
```

Comment:
```suggestion
  - **Understanding `ocamlnet`:** The `ocamlnet` provides a rich set of networking libraries that cover many protocols. It has some modules with a rich set of parameters. But for simple needs, the `Nethttp_client.Convenience` gives simple functions for basic interaction with the server. Note: These functions don't handle redirections.
```
Suggestion to avoid the use of "provide" multiple times